jkind/src/jkind/NullSettings.java

```diff
@@ -0,0 +1,8 @@
+package jkind;
+
+public class NullSettings extends Settings {
+
+	public static NullSettings instance() {
+		return new NullSettings();
+	}
+}
```

Comment:
Why have this class at all? You can just say `new Settings()` when you need this.